spread.yaml

```diff
@@ -1,21 +1,29 @@
 project: snapd
 
 environment:
-    REUSE_PROJECT: $(echo $REUSE_PROJECT)
+    REUSE_PROJECT: "$(HOST: echo $REUSE_PROJECT)"
     PROJECT_PATH: /home/gopath/src/github.com/snapcore/snapd
     GOPATH: /home/gopath
     PATH: /home/gopath/bin:/snap/bin:$PATH
-    TESTSLIB: $[PROJECT_PATH]/tests/lib
+    TESTSLIB: /home/gopath/src/github.com/snapcore/snapd/tests/lib
```

Comment:
Absolutely, thanks! :) I've taken the opportunity to reuse `$GOPATH` too, please take a look